cloudos/__main__.py

```diff
@@ -148,7 +153,7 @@ def run(apikey,
                 if j_status_h != j_status_h_old:
                     print(f'\tYour current job status is: {j_status_h}.')
                     j_status_h_old = j_status_h
-                time.sleep(1)
+                time.sleep(60)
```

Comment:
Is this something worth settings via a flag/param?

Reply:
I think is not necessary to change it. I extended as I've experiences some API instabilities in the past and reducing the number of API calls helped. Of course, it can be parametrised and I can rise a new issue as a reminder :+1: